Mathlib/RingTheory/MvPowerSeries/Basic.lean

```diff
@@ -154,6 +157,10 @@ theorem coeff_comp_monomial (n : σ →₀ ℕ) : (coeff R n).comp (monomial R n
 theorem coeff_zero (n : σ →₀ ℕ) : coeff R n (0 : MvPowerSeries σ R) = 0 :=
   rfl
 
+theorem ne_zero_iff_exists_coeff_ne_zero (f : MvPowerSeries σ R) :
```

Comment:
Do you want to also add the `f = 0` version? (Then this one could probably be `by simpa using f.eq_zero_….not`

Reply:
added. It is just "ext_iff".